pkg/plugins/models.go

```diff
@@ -82,7 +82,7 @@ func (pb *PluginBase) registerPlugin(base *PluginBase) error {
 	}
 
 	if !strings.HasPrefix(base.PluginDir, setting.StaticRootPath) {
-		plog.Info("Registering plugin", "name", pb.Name)
+		plog.Info("Registering plugin", "name", pb.Id)
```

Comment:
Labeling the ID as "name" looks wrong:
```suggestion
		plog.Info("Registering plugin", "id", pb.Id)
```